2.7.0/todo/result_intel.md

```diff
@@ -30,12 +30,18 @@ The categories below are as follows:
 ### deprecation
 ### new features
 ### improvements
+- Upgrade submodule oneDNN to v3.7.1 ([#148293](https://github.com/pytorch/pytorch/pull/148293))
```

Comment:
```suggestion
- Upgrade submodule oneDNN to v3.7.1 (#148293)
```
Please follow the instruction on line 12 and change the link formats to be shorter--on the actual release notes GitHub will render them properly.

Reply:
Thanks for the comment and changed.